README.md

```diff
@@ -152,6 +152,15 @@ files on disk for intermediate processing which is suboptimal for most cases,
 although makes examination of incredibly large files possible. Some of the
 results are spurious so use it with a lower confidence interval.
 
+#### Description
+
+Description is a backend for `DESCRIPTION` files which are text files to store
+important R package metadata. It finds names of licenses in the License field of
+the text file. This parser sometimes cannot identify licenses due to the name
```

Comment:
This last chunk, please take out of here and put into the source comments.

Reply:
Fixed